File: topics/control-flow/examples/TimesTable.pas
// 
// Program: TimesTable.pas
// Displays the Times Table from 1 x n to 10 x n.
//
program TimesTable;

procedure Main();
var
    number: Integer = 0;
    i: Integer = 0;
begin
    WriteLn('Times Table');
    
    Write('Enter number: ');
    ReadLn(number);
    
    WriteLn('-----------------');
    
    i := 1;
    while i < 10 do
    begin
        WriteLn(' ', i,' x ', number, ' = ', i * number);
        i += 1;
    end;
    
    WriteLn('-----------------');
end;

begin
    Main();
end.

File: topics/control-flow/examples/times_table.c
/*
* Program: times_table.c
* Displays the Times Table from 1 x n to 10 x n.
*/

#include <stdio.h>

int main()
{
    int number = 0;
    int i;
    
    printf("Times Table\n");
    
    printf("Enter number: ");
    scanf("%d", &number);
    
    printf("-----------------\n");
    
    i = 1;
    while(i < 10)
    {
        printf(" %d x %d = %d\n", i, number, i * number);
        i++;
    }
    printf("-----------------\n");
    
    return 0;
}